ddclient.in

```diff
@@ -648,11 +648,12 @@ my %services = (
         'examples'   => \&nic_easydns_examples,
         'variables' => {
             %{$variables{'service-common-defaults'}},
-            'backupmx'     => setv(T_BOOL,  0, 1, 0,                     undef),
-            'min-interval' => setv(T_DELAY, 0, 0, interval('5m'),        0),
-            'mx'           => setv(T_OFQDN, 0, 1, '',                    undef),
-            'server'       => setv(T_FQDNP, 1, 0, 'members.easydns.com', undef),
-            'wildcard'     => setv(T_BOOL,  0, 1, 0,                     undef),
+            'backupmx'     => setv(T_BOOL,   0, 1, 0,                    undef),
+            'min-interval' => setv(T_DELAY,  0, 0, interval('5m'),       0),
+            'mx'           => setv(T_OFQDN,  0, 1, '',                   undef),
+            'server'       => setv(T_FQDNP,  1, 0, 'api.cp.easydns.com', undef),
+            'script'       => setv(T_STRING, 1, 1, '/dyn/generic.php',   undef),
```

Comment:
Why would you want to change this?

Reply:
In exactly this situation. As a percentage of all API changes, it is actually pretty often that endpoints get moved without being modified. For basic users, it's nice to just be able to modify your config when it does happen, instead of having to wait a couple of years for the next release.